src/services/management/myAPICredentialApi.ts

```diff
@@ -114,11 +114,11 @@ export class MyAPICredentialApi extends Service {
     }
 
     /**
-    * @summary Generate new client key for self
+    * @summary Generate a client key
     * @param requestOptions {@link IRequest.Options}
     * @return {@link GenerateClientKeyResponse }
     */
-    public async generateNewClientKeyForSelf(requestOptions?: IRequest.Options): Promise<GenerateClientKeyResponse> {
+    public async generateClientKey(requestOptions?: IRequest.Options): Promise<GenerateClientKeyResponse> {
```

Comment:
```suggestion
    public async generateClientKey(requestOptions?: IRequest.Options): Promise<GenerateClientKeyResponse> {
```
Breaking change

Reply:
as discussed, this is not used at the moment, so we can go ahead 👍